pkg/proxy/snapshot.go

```diff
@@ -18,7 +19,18 @@ func (p *Proxy) VolumeSnapshot(ctx context.Context, req *rpc.EngineVolumeSnapsho
 		Snapshot: &eptypes.VolumeSnapshotReply{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.